boards/sodaq-sff/Makefile.features

```diff
@@ -0,0 +1,5 @@
+CPU_MODEL = samd21g18a
+
+FEATURES_PROVIDED += periph_pwm
```

Comment:
There's no pwm configured for this board. So just drop the line or add the PWM config ;)

Reply:
I'll take it out. And add the PWM in a separate PR for all Sodaq boards.